### DISCUSSION TITLE
请教下我用的版本是3.2 怎么在创建k8s组件的时候，设置挂载目录，将宿主机的目录挂载到容器内，谢谢了

### DISCUSSION THREAD

**Discussion by PhilipWang34** [apache/dolphinscheduler#15560] [Q&A]
请教下我用的版本是3.2 怎么在创建k8s组件的时候，设置挂载目录，将宿主机的目录挂载到容器内，谢谢了

**Comment by RaymondFishWang** [comment]
pvc

**Reply by PhilipWang34** [reply]
能给一个实际的例子吗

**Reply by RaymondFishWang** [reply]
你如果会k8s，应该会创建PVC啊，helm里也可以设置
![image](https://github.com/apache/dolphinscheduler/assets/152674254/715a91d5-8b83-4c4c-9d20-0736c3a216e0)
